src/tests/bulkConcepts/addBulkConcepts.test.jsx

```diff
@@ -113,14 +113,15 @@ describe('Add Bulk Concepts', () => {
       </Provider>
     </MemoryRouter>);
     const input = wrapper.find('#search');
-    input.simulate('change', { target: { value: '' } });
+    input.simulate('change', { target: { value: 'Bronze Diabetes' } });
     input.simulate('keydown', { key: 'ArrowDown' });
-    input.simulate('keydown', { key: 'Enter' });
+    input.simulate('keydown', { keyCode: 13 });
```

Comment:
Didn't you create a constant for the above?

Reply:
I have updated this to use the constant I made for the key code.